verify-prefill/package.json

```diff
@@ -4,6 +4,6 @@
   "private": true,
   "dependencies": {
     "twilio": "^5.0.4",
-    "node-fetch": "^3.3.2"
+    "node-fetch": "^2.7.0"
```

Comment:
I intentionally downgraded this to import with common-js compatibility which is easier to test. the only reason we have this import is that the package isn't available in the twilio-node library yet and once it is we'll be able to remove this.